packages/core/helper-plugin/lib/src/components/GenericInput/tests/index.test.js

```diff
@@ -128,4 +129,67 @@ describe('GenericInput', () => {
       expect(container).toMatchSnapshot();
     });
   });
+
+  describe('datetime', () => {
+    test('renders the datetime picker with the correct value for date and time', async () => {
+      const user = userEvent.setup();
+      const onClear = jest.fn();
+      const onChange = jest.fn();
+      const { getByRole } = render(
+        <ComponentFixture
+          type="datetime"
+          name="datetime-picker"
+          intlLabel={{
+            id: 'label.test',
+            defaultMessage: 'datetime',
+          }}
+          value={null}
+          onChange={onChange}
+          onClear={onClear}
+        />
+      );
```

Comment:
What do you think about moving this into it's own `setupDateTime` function similar to the users so you can re-use it on both tests without duplication? This will also make it easier in the future to setup new tests ...

Reply:
ok done